client/src/components/TrackGroup/TrackGroup.tsx

```diff
@@ -250,23 +272,22 @@ function TrackGroup() {
               </div>
             </div>
           </div>
-          <div
-            className={css`
-              display: flex;
-              justify-content: space-between;
-              @media screen and (max-width: ${bp.small}px) {
-                flex-direction: column;
-              }
-            `}
-          >
-            <AboutWrapper>
+          <TrackgroupInfosWrapper>
+            <AboutWrapper
+              trackGroupAbout={trackGroupAbout}
+              trackGroupCredits={trackGroupCredits}
+            >
```

Comment:
Wouldn't this be better achieved by doing 

```suggestion
            trackGroupAbout && <AboutWrapper
              trackGroupAbout={trackGroupAbout}
            >
```

Reply:
Maybe ! I'm not sure I understand the new code ahah

Reply:
Instead of using CSS to hide a section, you just don't display the section if the string doesn't exist.